tests/unit/test_tables.py

```diff
@@ -0,0 +1,17 @@
+from databuilder import tables
+from databuilder.contracts import constraints
+from databuilder.module_utils import is_proper_subclass
+
+
+def test_all_required_classes_are_exported():
+    # If we add a new Constraint we don't want to forget to make it available via
+    # `databuilder.tables`
```

Comment:
It's not clear to me why we want to include the constraint classes in `databuilder.tables`.

Reply:
It's all the stuff you need for defining tables. It felt like a cleaner abstraction to me to have that all bundled together (and `databuilder.tables` felt like the most appropriate place). But I don't feel particularly strongly about it.

Reply:
Don't feel strongly either.